commandapi-platforms/commandapi-bukkit/commandapi-bukkit-core/src/main/java/dev/jorel/commandapi/CommandAPIBukkit.java

```diff
@@ -354,10 +358,15 @@ private String[] getUsageList(RegisteredCommand currentCommand) {
 
 	void updateHelpForCommands(List<RegisteredCommand> commands) {
 		Map<String, HelpTopic> helpTopicsToAdd = new HashMap<>();
+		List<String> namespacedCommandNames = new ArrayList<>();
```

Comment:
Order doesn't matter in this context, we could be using a `Set<>` instead.

Reply:
Alright, guess I'll change that real quick and then I assume I am good to merge this?

Reply:
Yup, you're good to merge this to `dev/dev`.